src/amo/components/StaticPages/About/index.js

```diff
@@ -0,0 +1,111 @@
+
+  render() {
+    const { i18n } = this.props;
+
+    const helpText = i18n.sprintf(i18n.gettext(`Help improve this website. It's open source, and you can file bugs and submit patches. <a href="%(url)s">GitHub</a> contains all of our current bugs, legacy bugs can still be found in Bugzilla.`),
+      { url: 'https://github.com/mozilla/olympia/issues' });
+    const blogText = i18n.sprintf(i18n.gettext(`Our <a href="%(url)s">Add-ons Blog</a> is regularly updated with information for both add-on enthusiasts and developers.`),
+      { url: 'http://blog.mozilla.com/addons/' });
+    const forumText = i18n.sprintf(i18n.gettext(`Our <a href="%(url)s">forums</a> are a good place to interact with the add-ons community and discuss upcoming changes to AMO.`),
+      { url: 'https://discourse.mozilla-community.org/c/add-ons' });
+
+    /* eslint-disable react/no-danger */
+    return (
+
+      <Card
+        className="AboutPage"
+        header={i18n.gettext('About Mozilla Add-ons')}
+      >
+        <section>
+          <h2>{i18n.gettext('What is this website?')}</h2>
+          <p>
+            {i18n.gettext(`addons.mozilla.org, commonly known as "AMO", is Mozilla's official site for add-ons to Mozilla software, such as Firefox, Thunderbird, and SeaMonkey. Add-ons let you add new features and change the way your browser or application works. Take a look around and explore the thousands of ways to customize the way you do things online.`)}
+          </p>
+        </section>
+        <section>
+          <h2>{i18n.gettext('Who creates these add-ons?')}</h2>
+          <p>
+            {i18n.gettext('The add-ons listed here have been created by thousands of developers from our community, ranging from individual hobbyists to large corporations. All publicly listed add-ons are reviewed by a team of editors before being released. Add-ons marked as Experimental have not been reviewed and should only be installed with caution.')}
+          </p>
+        </section>
+        <section>
+          <h2>{i18n.gettext(`How do I keep up with what's happening at AMO?`)}</h2>
+          <p>
+            {i18n.gettext('There are several ways to find out the latest news from the world of add-ons:')}
+          </p>
+          <ul>
+            <li dangerouslySetInnerHTML={sanitizeHTML(blogText, ['a'])} />
+            <li>
+              {i18n.gettext('We often post news, tips, and tricks to our Twitter account,')}&nbsp;
+              <a href={'http://twitter.com/mozamo'}>{i18n.gettext('mozamo')}</a>
+            </li>
+            <li dangerouslySetInnerHTML={sanitizeHTML(forumText, ['a'])} />
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('This sounds great! How can I get involved?')}</h2>
+          <p>{i18n.gettext(`There are plenty of ways to get involved. If you're on the technical side:`)}
+            <ul>
+              <li>
+                <Link to={'/developers'}>{i18n.gettext('Make your own add-on')}</Link>.&nbsp;
```

Comment:
This is probably one of the few places we can probably live with splitting up the translations since I don't think we can really mix jsx inside a gettext call.  If there are other ideas on how we could deal with that better they'd be welcomed.